packages/playback-core/src/hls.ts

```diff
@@ -1,9 +1,10 @@
-// @ts-ignore
-import HlsMin from 'hls.js/dist/hls.min.js';
+import Hls from 'hls.js';
```

Comment:
NOTE: Keeping the changes narrow for now for both expediency and just in case we need to revert anything. We can minimally remove the commented out code pre-merge if preferred.